web/modules/weather_blocks/src/Plugin/Block/Test/EndToEnd/00_EndToEndBase.php.test

```diff
@@ -0,0 +1,153 @@
+<?php
```

Comment:
Yeah, exactly. It's annoying! I think your approach was to remove the `.test` extension? That'd work too. Do you have a preference, because I don't. 😛